docs/src/main/tut/applicative.md

```diff
@@ -43,6 +43,6 @@ into the other context:
 Applicative functors are a generalization of Monads thus allowing to express 
 effectful computations into a pure functional way.
 
-Applicative functors are generally preferred to monads when the structure 
+Applicative functors are generally preferred to Monads when the structure 
```

Comment:
@non Ok, I changed it because I thought it should be consistent with the sentence above. And I changed this one to uppercase (instead of the one above to lowercase) since I thought it was most in line with the style that is used (compare e.g. with usage of "Applicative functors" on this page).

Do you agree that both should have the same spelling? I'm happy to change it to the spelling that you prefer, but thought it was worth explaining why I changed it.